sdk/monitor/monitor-opentelemetry-exporter/src/utils/spanUtils.ts

```diff
@@ -95,102 +115,148 @@ function createPropertiesFromSpan(span: ReadableSpan): [Properties, Measurements
-    data.data = url.href;
-
-    if (span.attributes[HTTP_METHOD]) {
-      data.name = `${span.attributes[HTTP_METHOD] as string} ${url.pathname}`;
+  // DB Dependency
+  else if (dbSystem) {
+    remoteDependencyData.type = String(dbSystem);
+    const dbStatement = span.attributes[SemanticAttributes.DB_STATEMENT];
+    if (dbStatement) {
+      remoteDependencyData.data = String(dbStatement);
+    }
+    let target = getTarget(span);
+    const dbName = span.attributes[SemanticAttributes.DB_NAME];
+    if (target) {
+      remoteDependencyData.target = dbName ? `${target}/${dbName}` : `${target}`;
+    } else {
+      remoteDependencyData.target = dbName ? `${dbName}` : `${dbSystem}`;
     }
   }
-
-  if (span.attributes[DB_STATEMENT]) {
-    data.name = String(span.attributes[DB_STATEMENT]);
-    data.data = String(span.attributes[DB_STATEMENT]);
-    data.type = "DB";
-    if (span.attributes[DB_NAME]) {
-      data.target = String(span.attributes[DB_NAME]);
+  // grpc Dependency
+  else if (rpcMethod) {
+    const grpcStatusCode = span.attributes[SemanticAttributes.RPC_GRPC_STATUS_CODE];
+    if (grpcStatusCode) {
+      remoteDependencyData.resultCode = String(grpcStatusCode);
+    }
+    let target = getTarget(span);
+    if (target) {
+      remoteDependencyData.target = `${target}`;
+    } else {
+      const rpcSystem = span.attributes[SemanticAttributes.RPC_SYSTEM];
+      if (rpcSystem) {
+        remoteDependencyData.target = String(rpcSystem);
+      }
     }
   }
-
-  return data;
+  return remoteDependencyData;
 }
 
 function createRequestData(span: ReadableSpan): RequestData {
-  const data: RequestData = {
+  const requestData: RequestData = {
     name: span.name,
     id: `|${span.spanContext().traceId}.${span.spanContext().spanId}.`,
-    success: span.status.code === SpanStatusCode.OK,
-    responseCode: String(span.status.code),
+    success: span.status.code != SpanStatusCode.ERROR,
+    responseCode: "0",
     duration: msToTimeSpan(hrTimeToMilliseconds(span.duration)),
-    version: 1,
+    version: 2,
     source: undefined
   };
-
-  if (span.attributes[HTTP_METHOD]) {
-    data.name = span.attributes[HTTP_METHOD] as string;
-
-    if (span.attributes[HTTP_STATUS_CODE]) {
-      data.responseCode = String(span.attributes[HTTP_STATUS_CODE]);
-    }
-
-    if (span.attributes[HTTP_URL]) {
-      data.url = span.attributes[HTTP_URL] as string;
-    }
-
-    if (span.attributes[HTTP_ROUTE]) {
-      data.name = `${span.attributes[HTTP_METHOD] as string} ${span.attributes[
-        HTTP_ROUTE
-      ] as string}`;
-    } else if (span.attributes[HTTP_URL]) {
-      const url = new URL(span.attributes[HTTP_URL] as string);
-      data.name = `${span.attributes[HTTP_METHOD] as string} ${url.pathname}`;
-    }
-  }
-
-  if (span.attributes[GRPC_STATUS_CODE]) {
-    data.responseCode = String(span.attributes[GRPC_STATUS_CODE]);
+  const httpUrl = span.attributes[SemanticAttributes.HTTP_URL];
+  if (httpUrl) {
```

Comment:
Some special logic for populating [httpUrl](https://github.com/microsoft/common-schema/blob/main/Mappings/OTelSemanticConvention.md#mapping-for-httpurl) for server spans.